bazaarvoice/emodb
# integrate maven enforcer plugin

## Github Issue #

-

## What Are We Doing Here?

We have many libraries that are including the same maven transitive dependencies. Frequently these can be different versions. In some cases, the older version gets packaged in the jar introducing class implementation not found when running from the war.

Maven has a dependency enforcer. When run with it, there are many errors that will need to be resolved, but it will probably be a good idea to add this as part of the build. It will fail builds if we have two different versions getting included into the build.

## How to Test and Verify

1. Check out this PR
2. Run Command `mvn clean verify`
3. Profit

## Risk

### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary

Dependencies upgrade could cause issues in runtime.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
